src/api/signals.py

```diff
@@ -300,3 +307,26 @@ def run_cr_management_validation(sender, instance, *args, **kwargs):
 @receiver(pre_save, sender=Site)
 def update_with_covariates(sender, instance, *args, **kwargs):
     update_site_covariates_in_thread(instance)
+
+
+@receiver(post_save, sender=FishFamily)
+@receiver(post_delete, sender=FishFamily)
+@receiver(post_save, sender=FishGenus)
+@receiver(post_delete, sender=FishGenus)
+@receiver(post_save, sender=FishSpecies)
+@receiver(post_delete, sender=FishSpecies)
+@receiver(post_save, sender=BenthicAttribute)
+@receiver(post_delete, sender=BenthicAttribute)
+def bust_revision_cache(sender, instance, *args, **kwargs):
+    if sender is FishSpecies:
+        print(f"sender: {sender}")
+        cache.delete(FISH_SPECIES_SOURCE_TYPE)
```

Comment:
Isn't this the wrong way around? If somebody edits/deletes a species, we need to recalc averages for genus and family, right? I'm wondering if we shouldn't just delete them all if the sender is any of them, to be safe. Will hardly ever happen.